tools/c7n_left/README.md

```diff
@@ -180,6 +183,30 @@ policies:
     resource: [aws_ebs_volume, aws_sqs_queue]
 ```
 
+### taggable filter
+
+A `taggable` filter is available that allows filtering to only resources that support tagging.
+
+In combination with resource wild card support, this allows using a single policy to enforce
+an organization's tag standards.
+
+```
+policies:
+ - name: check-tag-policy
+   resource: "terraform.aws*"
```

Comment:
People are going to want wild cards on other providers too now...

Reply:
wild card on resources, yes they do, its the most upvoted issue.